<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="showImgIn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>420</width>
      <height>280</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fftImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FFT and IFFT</string>
    </property>
   </widget>
   <widget class="QLabel" name="showImgOut">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>420</width>
      <height>280</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="inputLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>After Inverse FFT</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="showData">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="idealLowpass">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ideal Lowpass</string>
    </property>
   </widget>
   <widget class="QPushButton" name="idealHighpass">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ideal Highpass</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openImg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fftImg</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>fftImg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>idealLowpass</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>idealLowpass()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>idealHighpass</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>idealHighpass()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openImg()</slot>
  <slot>fftImg()</slot>
  <slot>ifftImg()</slot>
  <slot>idealLowpass()</slot>
  <slot>idealHighpass()</slot>
 </slots>
</ui>
